### PR TITLE
[COMPOSER] Update structure of project

### PR DESCRIPTION
**Pull request by MProG**

### CODE REVIEW

File: files/database.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: myapp_test
+
+production:
+  <<: *default
+  database: myapp_production
```
Comment (roman-dubrovsky) [inline files/database.yml:23]: @MProG  https://github.com/datarockets/rails_apps_composer/issues/23

```
 url: <%= ENV["DATABASE_URL"] %>
```

File: files/initializers/jbuilder.rb [hunk 2]
```diff
@@ -0,0 +1 @@
+Jbuilder.key_format(camelize: :lower)
```
Comment (roman-dubrovsky) [inline files/initializers/jbuilder.rb:1]: @MProG  add `.editorconfig` to this project or install https://github.com/datarockets/dotfiles

File: files/database.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  username: username
+  password: you_password
```
Comment (AleksSenkou) [inline files/database.yml:5]: @MProG what do you think about skipping password for development db. it only makes development harder

File: files/database.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  url: <%= ENV["DATABASE_URL"] %>
```
Comment (AleksSenkou) [inline files/database.yml]: @MProG we don't need `url` option here. It's needed only for `production` in order to have access to db via url and keep this url as env variable

File: templates/helpers.erb [hunk 5]
```diff
@@ -65,6 +67,11 @@ def copy_from(source, destination)
   end
 end
 
+def copy_from_file(source, destination)
+  copy_from("base_path + source", destination)
+end
+
+
```
Comment (AleksSenkou) [inline templates/helpers.erb]: @MProG useless empty line 😉

File: recipes/rails_datarockets_api.rb [hunk 6]
```diff
@@ -29,6 +29,21 @@
...
+%w(helpers assets).each do |folder_name|
+  remove_dir "app/#{folder_name}"
+end
+
+%w(public/404.html public/500.html public/422.html config/initializers/assets.rb).each do |file_name|
+  remove_file file_name
+end
+
+insert_into_file('config/application.rb', 'config.api_only = true', 
+  after: /class Application < Rails::Application.*\n/, force: false)
```
Comment (AleksSenkou) [inline recipes/rails_datarockets_api.rb:41]: @MProG wow, cool done 👍

File: files/database.yml (hunk 3 above)
Comment (MProG) [reply files/database.yml:5]: How harder? You should only one time create user.

**Pull request merged**